pkg/registration/status.go

```diff
@@ -13,12 +13,39 @@ const (
 	// System is not registered yet.
 	Unregistered
 
-	// System has an expired subscription.
-	Expired
+	// Set when an error occurred
```

Comment:
I removed `Expired` from the options since this information is not easily fetchable by our current API. Instead of a complicated implementation I added `Unknown` which is returned if there was an error.